packages/terra-functional-testing/src/terra-cli/wdio/index.js

```diff
@@ -134,7 +134,7 @@ const cli = {
             return [process.env.THEME];
           }
 
-          return ['terra-default-theme'];
+          return [undefined];
```

Comment:
by leaving this an array with undefined we can still use the looping logic in the test runner, but loop once with an undefined theme.